### DISCUSSION TITLE
"Unexpected error" raised for my metrics

### DISCUSSION THREAD

**Discussion by PhilipDeegan** [lowlighter/metrics#1350] [Q&A]
Going to the link: https://metrics.lecoq.io/insights?user=PhilipDeegan

Shows at the top "Unexpected error".

And it seems that some of my contributions are missing (nvidia/thrust for example), so it could be related.

Feel free to close if this is a known issue.

**Comment by lowlighter** [comment]
Hi 👋 !

Seems the unexpected error comes from the sponsors plugin:
<img src="https://user-images.githubusercontent.com/22963968/212795843-9923a1d5-d7a3-475e-aad2-832a3c223631.png" width="400">

Not sure exactly what's causing it, but it's probably trying to access some fields that the public token doesn't have access and the error isn't handled correctly. Some of the GitHub sponsors data is only accessible by the user themself for privacy reasons.
It'll be tracked in #1360

___

As for your missing contributions, are you talking about the "Notable contributions" section ?

**Comment by PhilipDeegan** [comment]
> It'll be tracked in https://github.com/lowlighter/metrics/issues/1360

Awesome.

> As for your missing contributions, are you talking about the "Notable contributions" section ?

Yes, I think it was showing up  there before if I'm not mistaken.
